src/map/instance.cpp

```diff
@@ -66,240 +82,251 @@ struct instance_db *instance_searchname_db(const char *instance_name) {
  * @param sd: Player data to attach
  * @param target: Target display type
  */
-void instance_getsd(unsigned short instance_id, struct map_session_data **sd, enum send_target *target) {
-	switch(instance_data[instance_id].mode) {
+void instance_getsd(uint16 instance_id, struct map_session_data **sd, enum send_target *target) {
```

Comment:
I don't know if this will impact a lot but it would be preferable to have a pointer reference instead a pointer to pointer, well imo at least.
struct map_session_data*& sd, instead struct map_session_data**.
Never too fan about double or triple pointer, not really seen the use neither..